shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -671,6 +660,23 @@ public void onMessage(Object message, BasicMessageChannel.Reply<Object> reply) {
             case "announce":
                 mOwner.announceForAccessibility((String) data.get("message"));
                 break;
+            case "longPress": {
+                Integer nodeId = (Integer) annotatedEvent.get("nodeId");
+                if (nodeId == null) {
+                    return;
+                }
+                AccessibilityEvent e = obtainAccessibilityEvent(nodeId, AccessibilityEvent.TYPE_VIEW_LONG_CLICKED);
```

Comment:
Could you just be calling `sendAccessibilityEvent(int virtualViewId, int eventType)` here and below?

Reply:
Done